OpenEdXMobile/res/values/dimens.xml

```diff
@@ -8,6 +8,13 @@
     <!-- Per Material Design Guidelines -->
     <dimen name="action_bar_icon_size">24dp</dimen>
 
+    <!-- Alert dialog default padding -->
+    <dimen name="alert_dialog_padding_width">22dp</dimen>
+    <dimen name="alert_dialog_padding_height">11dp</dimen>
```

Comment:
I noticed that AlertDialog doesn't doesn't pad to match the textview. Also, there is no way to find the textview to query the padding dynamically. I honestly can't remember the exact spot that I found these dimens - I think it was on stack overflow somewhere (I'll do some searching after submitting the changes). But they match up nicely.